source/src/cip/cipassembly.h

```diff
@@ -9,8 +9,8 @@
 #include "typedefs.h"
 #include "ciptypes.h"
 
-/** @brief Assembly Class Code */
-static const int kCipAssemblyClassCode = 0x04;
+/** @brief Assembly class id */
+static const EipUint16 kCipAssemblyClassId = 0x04u;
```

Comment:
That's a good hint, but better to use CipUint (Eip types are a leftover, which should be removed I think)

Reply:
It's fine that I now know that the Eip* types are deprecated now. I will use the Cip* types then.

Here also the comment needs to some attention and also must be reverted back to "Class Code"

Reply:
Thanks for the hint. Regarding the Eip types, I have probably some similar traps still left in the code and undocumented. I have to learn to better convey design decisions and fix some documentation. now that you are also working on the code :) Sorry for that!

Reply:
As an additional info, I am not sure why the EIP types were introduced by the original contributer, but I found that always hard to tell when to use CIP and when to use EIP types. Especially as stdint.h types are not widely available on all platforms, I see no reason for the EIP type to exist.

Reply:
It was my impression that the Eip* types were used for code that is used to describe data "on the wire" i. e. the EtherNet/IP wire protocol. And that on the other hand code that handles "wire independent" stuff like internal object and connection management was using the Cip* data types.

This was my interpretation of the code I've seen. Also the border between these two "worlds" can't be absolutely clear because some data from "the wire" are moved into the objects at last.